sdks/go/node/containerruntime/docker/imagePuller.go

```diff
@@ -66,7 +66,9 @@ func (ip _imagePuller) Pull(
 	imagePullCreds := containerCall.Image.PullCreds
 	containerID := containerCall.ContainerID
 
-	imagePullOptions := types.ImagePullOptions{}
+	imagePullOptions := types.ImagePullOptions{
+		Platform: "linux",
```

Comment:
Why is this assumed now?

Reply:
I noticed it was being mentioned in logs it was looking for an image with Platform "darwin" and falling back to "linux" so It seemed "more correct" to specify linux. I can't think of a case we wouldn't want it to be that.